README.md

```diff
@@ -706,6 +707,40 @@ Similarly, defining tables out-of-order is discouraged as well.
 [b]
 ```
 
+Likewise, a table implied by a dot-key chain finishes being defined at the
+same moment as the table it is inside. After that, it cannot be redefined.
+
+```toml
+[public-domain]
+authors.'Mary Shelley'."Frankenstein; or, the Modern Prometheus" = true
```

Comment:
Hmm... These examples feel *very* involved and exhibit multiple concepts in addition to what the example is for (unicode, case sensitivity and quoted strings vs implicit tables from dotted keys). I'd prefer that we simplify these to only show the interactions of implicit tables.

I'd suggest using something like `subject.math.class-size` or something similarly trivial, as the thing to design the examples around, instead of `authors.'紫式部'."The Tale of Genji"`.

An involved example might also make things more complicated for translators though I don't recall if any translation has modified the example text.

Reply:
OK, I can do a second pass on trimming things down.

re: translation ease, I actually have worked on localization projects before and proper names are actually a boon, precisely because there is no conceptual space in the target language "for" that thing. Rather, it is a proper name, so it should not be semantically modified. Instead, if you want to make things more convenient, you can offer a phonetic remapping into the target language, maybe switch around a name order, and call it "done", without having to make any too-hard decisions like "how do I translate these two differing conceptions of what 'size' even is?" If they're of actual people, then probably someone has already done the work and created a canonical example, and you can end with a simple copy-and-paste job (translating names of non-real people is more complex because Reasons).

This doesn't mean there aren't other good reasons to not go with the examples I contrived! Far from it. I just wanted to explain that detail, as long as we're talking about i18n.

Reply:
Here's my shot at this example:

```toml
[fruit]
apple.color = "red"
apple.taste.sweet = true

# [fruit.apple]  # INVALID
# [fruit.apple.taste]  # INVALID

[fruit.apple.texture]  # you can add sub-tables to implicitly defined tables
smooth = true
```

Reply:
Hmm... Lemme make an equivalent PR, since I think I can budget the time now and procrastinate on studying.